dev-python/sphinx-autoapi/sphinx-autoapi-1.8.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="A new approach to API documentation in Sphinx"
+HOMEPAGE="https://sphinx-autoapi.readthedocs.io/"
+SRC_URI="https://github.com/readthedocs/sphinx-autoapi/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+# pypi lacks docs/
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+RDEPEND="
+	dev-python/sphinx[${PYTHON_USEDEP}]
+	dev-python/astroid[${PYTHON_USEDEP}]
+"
```

Comment:
I would like to be wrong but I'm afraid we need
```
BDEPEND="test? ( $RDEPEND )"
```
Unless I'm missing that this is automatically applied.

"8.1 Dependency Classes" in PMS says src_test phase can count on DEPEND, BDEPEND deps only.
https://projects.gentoo.org/pms/8/pms.html

@thesamesam does that make sense to you?
If so, I may apply this change before merging.

Reply:
d_e_t does this for us